internal/slicer/report.go

```diff
@@ -0,0 +1,61 @@
+package slicer
+
+import (
+	"fmt"
+	"github.com/canonical/chisel/internal/fsutil"
+	"github.com/canonical/chisel/internal/setup"
+	"io/fs"
+	"sync"
```

Comment:
nit: typically we keep standard packages, other external packages and internal packages separated by blank lines.